build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -69,45 +69,20 @@ public static BuildTargetResult UpdateBuild(BuildTargetContext c)
 
         // Moving PrepareTargets.RestorePackages after PackagePkgProjects because managed code depends on the
```

Comment:
This comment needs to be re-worded.  And maybe moving RestorePackages back to PrepareTargets would be a good idea...